### PR TITLE
Remove Travis CI integration

### PR DESCRIPTION
**Pull request by ssbarnea**
All tests performed by Travis are already covered by other CI/CD
jobs, so we remove it to lower the overall entropy.

Fixes: #823

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -9,9 +9,7 @@ name = ansible-lint
 url = https://github.com/ansible/ansible-lint
 project_urls =
   Bug Tracker = https://github.com/ansible/ansible-lint/issues
-  CI: Azure = https://dev.azure.com/ansible/ansible-lint/_build?view=runs
```
Comment (webknjaz) [inline setup.cfg:12]: This should be in a separate PR + you forgot about readme.

File: pyproject.toml [hunk 2]
```diff
@@ -6,8 +6,3 @@ requires = [
   "wheel",
 ]
 build-backend = "setuptools.build_meta"
-
-# ATTENTION: the following section must be kept last in
-# `pyproject.toml` because Travis CI appends one line in
```
Comment (webknjaz) [inline pyproject.toml:11]: This mustn't be removed as it's mandatory for the current setuptools-scm setup. The comment will be changed to GHA.

File: ROADMAP.rst [hunk 3]
```diff
@@ -6,6 +6,5 @@
 - Add `matchvar` method to lint vars in vars.yml #354
 - Add support for include_tasks which currently are skipped #362
 - Consider adopting/absorbing ansible-lint-junit #396
-- Use an API token for uploading to PyPI from Travis CI instead of account creds #553
```
Comment (webknjaz) [inline ROADMAP.rst:9]: Just change Travis CI  to GHA. Will also need to close that issue once complete.

**Comment by ssbarnea** [comment]
Sure take-over, no hard feelings.

**Comment by ssbarnea** [comment]
Already done.
